client/remote_state_access.go

```diff
@@ -22,6 +22,10 @@ func (client *ApiClient) RemoteStateAccessConfiguration(environmentId string) (*
 }
 
 func (client *ApiClient) RemoteStateAccessConfigurationCreate(environmentId string, payload RemoteStateAccessConfigurationCreate) (*RemoteStateAccessConfiguration, error) {
+	if payload.AllowedProjectIds == nil {
```

Comment:
If an array is not created explicitly, "null" is passed. The API doesn't accept "null". "[]" is needed instead.